ckanext/ozwillo_theme/fanstatic/theme_location.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+/* theme_location
+ *
+ * This JavaScript module gets a geojson from osm given a location name
+ * TODO: Find how to manage properly the required field,
+ *       Manage the case when the user click nowhere else than validate after typing the location
+ *       For now, the request always fails, find why
```

Comment:
?

Reply:
Done in the last version, except for the "required" field.
For now a pop up appears and ask the user to fill the location field, but this is done nicely in ckan with fields highligths. I couldn't get it to work here though.
If you don't want any popup I can remove it and let users be able to create datasets with no location.

Reply:
Yes, I prefer not having a popup (and yes, a dataset may not have a location or, at least, we have to allow it).